subsys/logging/Kconfig

```diff
@@ -290,7 +290,8 @@ config LOG_CMDS
 config LOG_BACKEND_UART
 	bool "Enable UART backend"
 	depends on UART_CONSOLE
-	default y
+	default y if !SHELL_BACKEND_SERIAL
+	default n
```

Comment:
is it needed? by default each parameter is `n`

Reply:
removed